.circleci/config.yml

```
@@ -30,6 +30,47 @@ jobs:
           paths:
             - .venv
             - integreat_compass.egg-info
+  npm-install:
+    docker:
+      - image: "cimg/node:lts"
+    resource_class: small
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - npm-{{ checksum "package-lock.json" }}-v2
```

```suggestion
            - npm-{{ checksum "package-lock.json" }}-v1
```
Starting with v1 is probably fine :sweat_smile:

---

Oh, whoops :D